From 208b13119fc0f709935fc2fc13c0295b19edbbf5 Mon Sep 17 00:00:00 2001
From: Sergey Matyukevich <geomatsi@gmail.com>
Date: Sat, 20 May 2017 15:36:51 +0300
Subject: [PATCH] ARM: dts: orange-pi-zero: enable spidev

On orange-pi-zero board SPI1 pins are accessible via GPIO expansion port.
This patch enables spidev driver for SPI1.

Signed-off-by: Sergey Matyukevich <geomatsi@gmail.com>
[Dario: make the patch to be applied with fuzz factor 0]
Signed-off-by: Dario Binacchi <dario.binacchi@amarulasolutions.com>
---
 .../dts/allwinner/sun8i-h2-plus-orangepi-zero.dts     | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts b/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts
index 4012f8712cd7..3552eb312747 100644
--- a/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts
+++ b/arch/arm/boot/dts/allwinner/sun8i-h2-plus-orangepi-zero.dts
@@ -59,6 +59,7 @@ aliases {
 		/* ethernet0 is the H3 emac, defined in sun8i-h3.dtsi */
 		ethernet0 = &emac;
 		ethernet1 = &xr819;
+		spi1 = &spi1;
 	};
 
 	chosen {
@@ -182,6 +183,16 @@ partition@00000000 {
 	};
 };
 
+&spi1 {
+	status = "okay";
+
+	spidev@0 {
+		compatible = "rohm,dh2228fv";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+	};
+};
+
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_pa_pins>;
-- 
2.43.0

